### PR TITLE
bdw-gc Recipe

### PR DESCRIPTION
**Pull request by stefan-balke**
Will fail since libatomic_ops is missing.

- [x] libatomic_ops (pending...)

### CODE REVIEW

File: recipes/bdw-gc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  build:
+    - autoconf
+    - automake
+    - libtool
+    - pkg-config
+    - libatomic_ops
+
+test:
+  commands:
+    - true
```
Comment (ocefpaf) [inline recipes/bdw-gc/meta.yaml]: Let's try to avoid this. Is there a command to test? A library to check for existence? (Or both?)

**Comment by ocefpaf** [comment]
> Will fail since libatomic_ops is missing.

I guess this will hold the rest of your PRs. Will fix that soon!

**Comment by jakirkham** [comment]
> I guess this will hold the rest of your PRs. Will fix that soon!

Deploying `libatomic_ops` as we speak.

**Comment by ocefpaf** [comment]
Thanks @stefan-balke! Just a few more to go :wink:

**Pull request merged**